wavelength/urls.py

```diff
@@ -16,6 +16,8 @@
 from django.contrib import admin
 from django.urls import include, path
 
+app_name="app"
```

Comment:
Remove this line. It shouldn't be needed because we are not using "name='whatever_name'" on the paths.